solr/solrj/src/test/org/apache/solr/client/solrj/impl/Http2SolrClientTest.java

```diff
@@ -299,7 +299,9 @@ public void testAsyncException() throws Exception {
   public void testAsyncAndCancel() throws Exception {
     String url = getBaseUrl() + DEBUG_SERVLET_PATH;
     Http2SolrClient.Builder b =
-        new Http2SolrClient.Builder(url).withResponseParser(new XMLResponseParser());
+        new Http2SolrClient.Builder(url)
+            .withConnectionTimeout(DEFAULT_CONNECTION_TIMEOUT, TimeUnit.MILLISECONDS)
```

Comment:
Not sure if this is out of scope; tied to the other thing you agreed should be separate.  A minor matter anyway.

Reply:
The reason I had added the null check to the connection timeout was because if the test does not explicitly set it, the user gets a NPE.  I removed it because I agree it is out of scope, but that means I need to explicitly set it in the test, as all prior tests had done.